blocked-edges/4.11.58-OldKernelHighLoadIOWait.yaml

```diff
@@ -0,0 +1,8 @@
+to: 4.11.58
+from: .*
+
```

Comment:
We do not need the extra new line here.

Reply:
83f52a86e8818d6524261483817dfe64547d5a91 does not have this blank line.